Source Code/Ticket Management System/supportgiver_menu.c

```diff
@@ -77,143 +77,206 @@ bool searchByTags(void* ticket, void* tag)
 	return found_tag;
 }
 
-void browseTickets()
+void filterByStatusAction() 
 {
-	wprintf(L"Browse Tickets\n");
-	wprintf(L"==============\n");
 	struct Vector* tickets = ticketContainer_getAll();
-	if (tickets != NULL)
+	if (tickets == NULL) {
+		wprintf(L"No tickets in the system.\n");
+		return 0;
```

Comment:
Returning `0` from void?
Also, there's no need to return because the other stuff are in an `else`.